Makefile

```diff
@@ -0,0 +1,28 @@
+all: test
+
+PACKAGES := $(shell go list ./... | grep -v vendor)
+
+init:
+	go get -u github.com/AlekSi/gocoverutil
+	go get -u gopkg.in/alecthomas/gometalinter.v1
+	gometalinter.v1 --install
+
+install:
+	go install -v $(PACKAGES)
+	go test -v -i $(PACKAGES)
+
+install-race:
+	go install -v -race $(PACKAGES)
+	go test -v -race -i $(PACKAGES)
```

Comment:
I don't like pushing everywhere `go test`, since it's `install` I would expect only `go install`. But do as you wish.